modules/database/src/ioc/rsrv/rsrvIocRegister.c

```diff
@@ -20,7 +20,11 @@
 /* casr */
 static const iocshArg casrArg0 = { "level",iocshArgInt};
 static const iocshArg * const casrArgs[1] = {&casrArg0};
-static const iocshFuncDef casrFuncDef = {"casr",1,casrArgs};
+static const iocshFuncDef casrFuncDef = {"casr",1,casrArgs,
+                                         "Channel Access Server Report with following levels:\n"
+                                         "  0 - server’s protocol version level and summary for each attached client\n"
+                                         "  1 - extends report with information about connected clients and beacons\n"
+                                         "  2 - extends report with specific channel names and UDP search requests\n"};
```

Comment:
fyi. There are higher levels up detail, up to 10 I think.  The output is highly expert friendly.  I'm not sure how to express this.  Maybe `  3+ - expert`?  Maybe not worth mentioning?

Reply:
expanded in a271e4dca11964a0b899b33b8811451e2308b867